src/api/api.cpp

```diff
@@ -71,7 +73,8 @@ migraphx_status try_(F f, bool output = true) // NOLINT
         catch(const migraphx::exception& ex)
         {
             if(output)
-                std::cerr << "MIGraphX Error: " << ex.what() << std::endl;
+                std::cerr << "MIGraphX Error: Caller:" << llc.function_name()
```

Comment:
I would put the function name first: `std::cerr << llc.function_name() << ": Error: " << ex.what() << std::endl"`, and we can probably remove the `MIGraphX` since it will be in the function name.